README.md

```diff
@@ -2,16 +2,40 @@
 
 [![Build Status](https://travis-ci.org/hmrc/api-subscription-fields.svg)](https://travis-ci.org/hmrc/api-subscription-fields) [ ![Download](https://api.bintray.com/packages/hmrc/releases/api-subscription-fields/images/download.svg) ](https://bintray.com/hmrc/releases/api-subscription-fields/_latestVersion)
 
-This microservice stores definitions and values for the HMRC Developer Hub.
+This microservice stores and retrieves subscription fields and their values.
 
+### Fields Definition Endpoint examples
 
-### Tests
+#### Add or modify fields definition record
+
+
+#### Get fields definition by api context and api version
+
```

Comment:
add example

Reply:
Added.

Reply:
Just realised that the fields definitions are not implemented in the routes and controller yet, so I won't be able to test it...